src/containers/Toolbar/index.tsx

```diff
@@ -10,19 +11,34 @@ interface IToolbarProps {
   onFilterValueChange: (filterValue: string) => void;
   preserveLogs: boolean;
   onPreserveLogsChange: (preserveLogs: boolean) => void;
+  onClear: () => void;
 }
 
+const ClearButton = ({ onClick }: { onClick: () => void }) => (
+  <div className="border-0 rounded-md p-2 opacity-50 outline-none hover:opacity-100">
```

Comment:
I don't think any of these styles are needed. Button styling is all covered inside IconButton. I think all we need is to make sure the button remains centered without adding extra padding.

```
className="flex items-center"
```

This will produce a result like:
<img width="495" alt="Screenshot 2021-09-21 at 12 53 19" src="https://user-images.githubusercontent.com/2871038/134165806-3b88024e-5787-4552-b90f-a8bccf5f335f.png">
<img width="527" alt="Screenshot 2021-09-21 at 12 53 54" src="https://user-images.githubusercontent.com/2871038/134165879-d35d010e-959a-46c4-8ed1-d65622a71b52.png">